src/magic_folder/api_cli.py

```diff
@@ -9,6 +9,8 @@
 import json
 from collections import deque
 
+import humanize
```

Comment:
If we are going to use this directly, we should explicitly depend on it.

Reply:
is there any documentation for `requirements/*`?

Reply:
No, sorry. I guess https://pip-compile-multi.readthedocs.io/en/latest/boilerplate.html is probably close, although the command to run is `tox -e pin-requirements`. You can also add `-- --no-upgrade ` to that, to only resolve new dependencies, not update any other dependencies.

Reply:
okay i added instructions and a req.